keyboards/orthodox/keymaps/rfvizarra/keymap.c

```diff
@@ -0,0 +1,206 @@
+    KC_DEL,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,            _______, _______,          _______, _______,          KC_F6,   KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS, \
+    _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  _______, _______, _______,          _______, _______, _______, KC_F12,  KC_PGUP, KC_COMM, KC_DOT,  KC_PGDN, _______ \
+  ),
+
+  [_NAV] =  LAYOUT( \
+    _______, _______, _______, _______, _______, _______,                                                                _______, _______, KC_UP,   _______,  _______, _______, \
+    _______, _______, _______, _______, _______, _______,          _______, _______,          _______, _______,          _______, KC_LEFT, KC_DOWN, KC_RIGHT, _______, _______, \
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______,  _______, _______ \
+  ),
+
+  [_NAV2] =  LAYOUT( \
+    _______, _______, _______, _______, _______, _______,                                                                _______, _______, KC_PGUP, _______, _______, _______, \
+    _______, _______, _______, _______, _______, _______,          _______, _______,          _______, _______,          _______, KC_HOME, KC_PGDN, KC_END,  _______, _______, \
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______ \
+  ),
+
+  [_MEDIA] =  LAYOUT( \
+    _______, _______, _______, _______, _______, _______,                                                                _______, _______, KC_VOLU, _______, _______, _______, \
+    _______, _______, _______, _______, _______, _______,          _______, _______,          _______, _______,          _______, KC_MNXT, KC_VOLD, KC_MPLY, _______, _______, \
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______ \
+  ),
+
+  [_ADJUST] =  LAYOUT(                                                  \
+    _______, RESET,   _______, _______, _______, _______,                                                                _______, _______, _______, _______, _______, _______,  \
+    _______, _______, _______, AU_ON,   AU_OFF,  AG_NORM,          _______, _______,          _______, _______,          AG_SWAP, QWERTY , COLEMAK, DVORAK,  _______, _______, \
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY  \
+  )
+
+
+};
+
+#ifdef AUDIO_ENABLE
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+float tone_dvorak[][2]     = SONG(DVORAK_SOUND);
+float tone_colemak[][2]    = SONG(COLEMAK_SOUND);
+#endif
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
Not needed.  Jack added this as a core function: `set_single_persistent_default_layer`
https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/keymaps/default/keymap.c#L190

This looks  a bit cleaner (you only need to add the layer number/label).  And it handles the audio if you define it, like this: 
```c
    #define DEFAULT_LAYER_SONGS { SONG(QWERTY_SOUND), \
                                  SONG(COLEMAK_SOUND), \
                                  SONG(DVORAK_SOUND) \
                                }
```
https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/keymaps/default/config.h#L10

(in your config.h file)